ontotext-yasgui-web-component/README.md

```diff
@@ -105,6 +105,11 @@ The "config" value of "ngce-prop-config" or "[config]" is an object with followi
 - **pageSize**: the size of a page. Default value is 10.
 - **downloadAsOn**: if false "Download as" dropdown will not be shown.
 - **yasrToolbarPlugins**: "ontotext-yasgui-web-component" has a toolbar on the right next to the plugin buttons. **yasrToolbarPlugins** have to contain elements that implements [YasrToolbarPlugin](src/models/yasr-toolbar-plugin.ts) interface.
+- **readonly**: There are three options:
+   - true - the query can be edited;
+   - false - the editor is read-only, but the query can be copied;
+   - 'nocursor' - the editor is read-only and hte query can't be copied.
```

Comment:
If this `readonly` property can have this `nocursor` string value, then it's not a boolean type anymore and the other options should be `true` and `false` which is not good approach. IMO we should either split it to two separate config properties or to change these true/false to some other string values.

Reply:
What you think about "yasqeMode" with three options: "WRITE", "READ", "READ_WITH_COPY"

Reply:
What about `READ_WITH_COPY` -> `PROTECTED`?

Reply:
Done